opentelemetry-api/tests/baggage/test_baggage_propagation.py

```diff
@@ -154,3 +157,12 @@ def test_fields(self, mock_format_baggage, mock_baggage):
             inject_fields.add(mock_call[1][1])
 
         self.assertEqual(inject_fields, self.propagator.fields)
+
+    def test__format_baggage(self):
+        self.assertEqual(
+            _format_baggage({"key key": "value value"}), "key+key=value+value"
+        )
+        self.assertEqual(
+            _format_baggage({"key/key": "value/value"}),
+            "key%2Fkey=value%2Fvalue",
+        )
```

Comment:
Can you also add a `extract` test with these special characters?

Reply:
fixed